index.html

```diff
@@ -1,9 +1,20 @@
 <!doctype html>
 <html class="no-js" lang="">
 <head>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
```

Comment:
You definitely don't need JS for the grid. I've pushed up [where I got to before I read you were keen to do the player card](https://github.com/pjho/Elo-Score-Board/tree/player-card-eg). That branch has the almost full version of bootstrap css included in the main style sheet & no js & the grid is working fine.

While loading static assets from a public cdn has the advantage that if a user has visited a different site with the same asset previously it may be cached in their browser, we also want to have as few http requests as possible. So there is always some trade off in FE optimisation. In this instance I would bundle the css into our own single stylesheet. jQuery I would probably load from cdn but would reference it in the footer (not that we display anything before scripts load anyway but still a good practice). Bootstrap JS I would bundle. However I'm still adamant we don't need the JS so would prefer to remove this block.

Also our users are almost 100% returning users, so we will have the benefit of cached assets in any case...